tests/unit/code_provider/test_git_safe.py

```
@@ -0,0 +1,574 @@
+        with patch(
+            "app.modules.code_provider.git_safe.safe_git_operation"
+        ) as mock_safe_op:
+            mock_safe_op.return_value = "result"
+
+            result = safe_git_repo_operation(
+                "/some/path", mock_operation, max_retries=3, timeout=30.0
+            )
+
+        assert result == "result"
+        mock_safe_op.assert_called_once()
+        args, kwargs = mock_safe_op.call_args
```

Ruff (0.15.10)

[warning] Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)